### PR TITLE
Fix releasever for RHEL8

### PR DESCRIPTION
**Pull request by lhellebr**
Fixes https://github.com/SatelliteQE/robottelo/issues/9403

### CODE REVIEW

File: pytest_fixtures/component/subscription.py [hunk 1]
```diff
@@ -12,12 +12,16 @@ def subscribe_satellite(clean_rhsm, default_sat):
...
     default_sat.register_contenthost(
         org=None,
         lce=None,
         username=settings.subscription.rhn_username,
         password=settings.subscription.rhn_password,
-        releasever=f'{default_sat.os_version.major}Server',
+        releasever=release_version,
     )
     result = default_sat.subscription_manager_attach_pool([settings.subscription.rhn_poolid])[0]
     if 'Successfully attached a subscription' in result.stdout:
```
Comment (Gauravtalreja1) [inline pytest_fixtures/component/subscription.py:27]: I guess a repo label like `rhel-8-server-extras-rpms` doesn't exist in CDN, are you planning to change that as well?

File: pytest_fixtures/component/subscription.py [hunk 2]
```diff
@@ -12,12 +12,16 @@ def subscribe_satellite(clean_rhsm, default_sat):
     """subscribe satellite to cdn"""
     from robottelo.config import settings
 
+    if default_sat.os_version.major < 8:
+        release_version = f'{default_sat.os_version.major}Server'
+    else:
+        release_version = f'{default_sat.os_version.major}'
```
Comment (Gauravtalreja1) [inline pytest_fixtures/component/subscription.py:18]: Optional: Changing this to one-liner would be nice, like,
```suggestion
    major_version = default_sat.os_version.major
    release_version = f'{major_version}Server' if major_version < 8 else f'{major_version}'
```

**Review by jyejare — APPROVED** [review]

**Review by mshriver — APPROVED** [review]

File: pytest_fixtures/component/subscription.py (hunk 1 above)
Comment (mshriver) [reply pytest_fixtures/component/subscription.py:27]: @lhellebr Please respond here - non-blocking for the diff for this commit, but might need addressed.

Comment (lhellebr) [reply pytest_fixtures/component/subscription.py:27]: https://github.com/SatelliteQE/robottelo/pull/9654

**Pull request merged**